cmd/operator/main.go

```diff
@@ -100,18 +101,23 @@ func addReconcilersToManager(mgr manager.Manager, restCfg *rest.Config) {
 		return
 	}
 
+	cacheManager := vcache.MakeCacheManager(opcfg.GetIsCacheEnabled())
+	if !opcfg.GetIsCacheEnabled() {
```

Comment:
main() already logs all options, perhaps you can add this  there so it is not isolated